views/networking/load_balancer/show.erb

```diff
@@ -26,10 +26,10 @@
         ["Connection String", @lb[:hostname], { copieble: true }],
         [
           "Private Subnet",
-          "<a href='#{@project_data[:path]}#{@lb[:subnet][:path]}' class='text-rose-500 hover:underline'>#{@lb[:subnet][:name]}</a>",
+          "<a href='#{@project_data[:path]}/location/#{@lb[:location]}/private-subnet/#{@lb[:subnet]}' class='text-rose-500 hover:underline'>#{@lb[:subnet]}</a>",
           { escape: false }
         ],
-        ["Algorithm", @lb[:algorithm]],
+        ["Algorithm", (@lb[:algorithm] == "round_robin") ? "Round Robin" : "Hash Based"],
```

Comment:
in the provisioning drop down, it's called "round-robin" and "hash-based". Wondering if we should call it the same everywhere, maybe "Round-robin" and "Hash-based".

Reply:
I have updated the create experience to have "Round Robin" and "Hash Based"